packages/core/src/lib/light_push/light_push.ts

```diff
@@ -67,7 +69,13 @@ export class LightPushCore {
         };
       }
 
-      const query = PushRpc.createRequest(protoMessage, encoder.pubsubTopic);
+      const pubsubTopic = determinePubsubTopic(
```

Comment:
Now, we determine the pubsub topic when sending the message, so we have context (cluster id) to compute it for auto-sharding.